src/components/Pill/index.tsx

```diff
@@ -41,7 +41,7 @@ export const Container = styled.span<PillProps>`
       : p.variant === 'warning'
       ? color.warning
       : p.variant === 'disabled'
-      ? color.inactive
+      ? color.strokeStrong
```

Comment:
Hmm I think this should be `inactiveBackground` instead 🤔  and then `onInactive` would probably look better already?